src/components/twilio-voice-dialer/public/twilio-voice-dialer.js

```diff
@@ -1 +1,51 @@
-// TODO: Implement Web Component
+const render = (x) => `
+  <div class="container">
+    <p id="status">Status: pending</p>
+    <input type="text" placeholder="recipient" id="recipient" value=${x.recipient}>
+  </div>
+`;
+
+class TwilioVoiceDialer extends HTMLElement {
+  static get observedAttributes() {
+    return ['recipient'];
+  }
+
+  #device;
+  #token;
+
+  constructor() {
+    super();
+    this.attachShadow({ mode: 'open' });
+  }
+  get recipient() {
+    return this.getAttribute('recipient');
+  }
```

Comment:
Why do we need a getter?

Reply:
We need the getter to set the `value=${x.recipient}` in the `input` html https://github.com/twilio/twilio-voice-js-reference-components/pull/2/files#diff-731372f63bb700410eff73a4a78531255e1a21231965fc053c640d4314f6dbbaR4

Reply:
Since it's only used for internal, no need to expose this publicly. We're not really saving any lines of code. We can call this.getAttribute directly.